auth-api/src/auth_api/models/org.py

```diff
@@ -240,3 +244,25 @@ def reset(self):
             self.save()
         else:
             super().reset()
+
+
+@event.listens_for(Org, 'before_insert')
+def receive_before_insert(mapper, connection, target):  # pylint: disable=unused-argument; SQLAlchemy callback signature
+    """Rejects invalid type_codes on insert."""
+    org = target
+    if org.type_code in (OrgTypeEnum.SBC_STAFF.value, OrgTypeEnum.STAFF.value):
+        raise BusinessException(
+            Error.INVALID_INPUT,
+            None
+        )
+
+
+@event.listens_for(Org, 'before_update', raw=True)
```

Comment:
you can stack this with the 'before_insert' so that you don't have duplicate code unless there's a reason you want to separate these

Reply:
`raw=True` is the reason why they're separate - changes the object "state"

Reply:
They have to be seperate